dump25/claims_tabs/tab4.py

```
@@ -0,0 +1,229 @@
+            for line in ujson.load(f):
+                yield line
+
+    def read_file(self):
+        # ---
+        current_count = self.current_count
+        # ---
+        xn = 0
+        # ---
+        for x in tqdm.tqdm(self.jsonfiles):
+            # ---
+            xn += 1
```

Ruff (0.8.2)

Use `enumerate()` for index variable `xn` in `for` loop

(SIM113)